funnel/templates/403.html.jinja2

```diff
@@ -0,0 +1,13 @@
+{% extends "layout.html.jinja2" %}
+{% block title %}403 Forbidden{% endblock %}
+
+{% block content %}
+  <div>
+    <img src="{{ url_for('static', filename='img/error-403.svg') }}" class="emoticon"/>
+    <p class="mui--text-light mui--text-subhead">{% trans %}Unauthorized{% endtrans %}</p>
```

Comment:
Let's call this "Unavailable" because "unauthorized" is (a) too formal, and (b) sometimes inaccurate as we use 403 for unavailable transitions. It makes it seem like the user has no access to their own content, instead of telling the user that an action is not available.